docs/reference/webapp/explore_records.md

```diff
@@ -53,6 +53,6 @@ If you have multiple predictions you can toggle between them using the arrows on
 
 ## Metrics
 
-_TODO: Screenshot_
+![Metrics for Explore Mode](../../_static/reference/webapp/metrics_exploremode.png)
```

Comment:
@leireropl This SS should show the "Stats" metrics, the "Progress" is better suited for the annotation mode.

Reply:
Done!